pkg/agent/controller/traceflow/packetin.go

```diff
@@ -101,13 +101,13 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*opsv1alpha1.Tracefl
 
 	if isSender {
 		ob := new(opsv1alpha1.Observation)
-		ob.Component = opsv1alpha1.SpoofGuard
-		ob.Action = opsv1alpha1.Forwarded
+		ob.Component = opsv1alpha1.ComponentSpoofGuard
```

Comment:
Probably we missed something - are using this opportunity to refactor APIs? If not why such renaming?
@tnqn

Reply:
Name collision I assume. Everything is in the same API group / package now

Reply:
It doesn't affect APIs, this is the naming of the Go constants, the underlying string remains the same

Reply:
I understood, but it will create lots of conflicts with other PRs. Also I feel much easier to track the change with a separate PR. But anyway, since it is done..

Reply:
This patch adds "Component" because "NetworkPolicy" is the name of a struct too, seems no better way than adding prefix to `TraceflowComponent` constant `NetworkPolicy`

Reply:
Ok. Then makes sense to me. Agree TraceflowComponent is better, but to merge this one earlier, we can do another refactoring later.